django_tables2/export/export.py

```diff
@@ -41,16 +41,17 @@ class TableExport:
         YAML: "text/yml; charset=utf-8",
     }
 
-    def __init__(self, export_format, table, exclude_columns=None):
+    def __init__(self, export_format, table, exclude_columns=None, dataset_kwargs=None):
```

Comment:
Can you add the `dataset_kwargs` arugment to the docstring (row 21 of this file)?

Reply:
Oops!  Done.